README.md

```diff
@@ -48,9 +48,9 @@ A mobile web application where users can manage their budget: they have a list o
 ## 🚀 Live Demo <a name="live-demo"></a>
 
 
-## [Go live]()
+## [Go live](https://smart-budget-x8t8.onrender.com)
 
-## [Video presentation]()
+## [Video presentation](https://www.loom.com/share/b6dd8f60f9e44dbdacca9220c485e714)
```

Comment:
- [ ] As suggested by the first reviewer. Kindly record a new presentation video and mention a piece of code you are most proud of in the video. This is part of the project requirements.
![image](https://github.com/aubinleyoung/Budget-app/assets/94551484/e998df00-8b09-43e4-a1c2-f5a12e1ea478)